conda/cli/conda_argparse.py

```diff
@@ -361,15 +361,18 @@ def configure_parser_info(sub_parsers):
         description=help,
         help=help,
     )
-    add_parser_json(p)
+    add_parser_json(p, include_verbosity_option=False)
     p.add_argument(
         "--offline",
         action='store_true',
         default=NULL,
         help=SUPPRESS,
     )
     p.add_argument(
-        '-a', "--all",
+        "-a",
+        "--all",
+        "-v",
+        "--verbose",
```

Comment:
The problem is that `verbosity` is also a config setting (.condarc etc) and I don't think this change will appropriately handle the case of loading it from there, as it's not using `dest="verbosity"`.

I would suggest to apply this:
```suggestion
        "-a",
        "--all",
        "-v",
        "--verbose",
        dest="verbosity",
```

and then update the CLI handler (`main_info.execute`) to use `args.verbosity` instead of `args.all`.